app/src/main/java/com/github/orkest/View/FollowListActivity.kt

```diff
@@ -0,0 +1,220 @@
+    ) {
+        Image(
+            painter = painterResource(user.profile.profilePictureId),
+            contentDescription = "Contact profile picture",
+            modifier = Modifier
+                .size(itemSize)
+                .clip(CircleShape)
+                .padding(start = 10.dp)
+        )
+
+        Spacer(modifier = Modifier.width(smallSeparator))
+
+        Column (
+            modifier = Modifier.height(itemSize),
+            horizontalAlignment = Alignment.Start,
+            verticalArrangement = Arrangement.Center) {
+            Text(
+                text = user.username,
+                style = TextStyle(fontSize = textFontSize, fontWeight = FontWeight.Bold)
+            )
+        }
+    }
+}
+
+
+/**
+@Composable
+fun FollowListScreen(viewModel: FollowListViewModel) {
+    var userList by remember { mutableStateOf(mutableListOf<User>()) }
+
+    viewModel.retrieveFollowList().thenAccept {
+        userList = it
+    }
+
+    Column{
+        val context = LocalContext.current
+        IconButton(onClick = {
+                val intent = Intent(context, ProfileActivity::class.java)
+                intent.putExtra("username", viewModel.username)
+                context.startActivity(intent)})
+        {
+            Image(
+                painterResource(id = R.drawable.back_button),
+                contentDescription = "Back button",
+                modifier = Modifier
+                    .padding(10.dp)
+                    .height(100.dp)
+                    .width(100.dp)
+            )
+        }
+        Spacer(modifier = Modifier.height(20.dp))
+
+        LazyColumn {
+            items(userList) { user ->
+                val context = LocalContext.current
+                val intent = Intent(context, ProfileActivity::class.java)
+                intent.putExtra("username",user.username)
+                UserListItem(user = user, intent = intent)
+            }
+        }
+    }
+    
+}
+
+@Composable
+fun UserList(users: MutableList<User>) {
+    LazyColumn {
+        items(users) { user ->
+            val context = LocalContext.current
+            val intent = Intent(context, ProfileActivity::class.java)
+            intent.putExtra("username",user.username)
+            UserListItem(user = user, intent = intent)
+        }
+    }
+}
+
+@Composable
+fun UserListItem(user: User, intent: Intent){
+    val context = LocalContext.current
+    Row(modifier = Modifier
+        .padding(all = 10.dp)
+        .clickable { context.startActivity(intent) }
+        .fillMaxSize()){
+
+        Image(
+            painter = painterResource(user.profile.profilePictureId),
+            contentDescription = "Profile picture",
+            modifier = Modifier
+                .size(40.dp)
+                .clip(CircleShape)
+        )
+
+        // Add a horizontal space between the image and the text
+        Spacer(modifier = Modifier.width(10.dp))
+
+        Text(text = user.username)
+    }
+}
+**/

```

Comment:
What is the purpose of the commented code ?

Reply:
This commented code contains the fetching from the database and update of the UI. As I have said earlier, I did not finish implementing the backend for this feature as it was not a task for this sprint.